packages/@okta/vuepress-site/docs/reference/api/devices/index.md

```diff
@@ -100,19 +120,23 @@ curl -v -X GET \
         "imei": null,
         "meid": null,
         "udid": null,
-        "sid": "S-1-11-111"
+        "sid": "S-1-11-111",
+        "tpmPublicKeyHash":null,
+        "registered":true,
+        "managed":false,
+        "secureHardwarePresent":false
     },
     "_links": {
         "activate": {
-            "href": "https://{yourOktaDomain}/api/v1/devices/guo4a5u7JHHhjXrMK0g4/lifecycle/activate",
+            "href": "https://${yourOktaDomain}/api/v1/devices/guo4a5u7JHHhjXrMK0g4/lifecycle/activate",
```

Comment:
We don't include `$` in the response samples. See https://github.com/okta/okta-developer-docs/wiki/Style-Guide#variables-in-examples
```suggestion
            "href": "https://{yourOktaDomain}/api/v1/devices/guo4a5u7JHHhjXrMK0g4/lifecycle/activate",
```